packages/frontend/src/components/common/Dashboard/DashboardHeader.tsx

```diff
@@ -294,13 +296,15 @@ const DashboardHeader = ({
 
             {userCanManageDashboard && isEditMode ? (
                 <PageActionsContainer>
-                    <AddTileButton
-                        onAddTiles={onAddTiles}
-                        disabled={isSaving}
-                        setAddingTab={setAddingTab}
-                        activeTabUuid={activeTabUuid}
-                        dashboardTabs={dashboardTabs}
-                    />
+                    {!hasNewSemanticLayerChart && (
+                        <AddTileButton
+                            onAddTiles={onAddTiles}
+                            disabled={isSaving}
+                            setAddingTab={setAddingTab}
+                            activeTabUuid={activeTabUuid}
+                            dashboardTabs={dashboardTabs}
+                        />
```

Comment:
We should disable the `+ Add new chart` button instead of the whole add tile:

![image](https://github.com/user-attachments/assets/dd307a1f-1b64-4b4b-b7c7-0daf76bcca30)

After https://github.com/lightdash/lightdash/issues/11726 the list there will only contain charts belonging to the same semantic layer

Reply:
omg! ... 😿